src/WalkingModule/app/robots/ergoCubGazeboV1/dcm_walking_with_joypad.ini

```diff
@@ -54,6 +54,9 @@ sampling_time           0.01
 # Specify the frame to use to control the robot height. Currently, we support only the following options: com, root_link
 height_reference_frame  root_link
 
+# Loop rate for the thread computing the navigation trigger
+navigationTriggerLoopRate   100
+
```

Comment:
As mentioned in https://github.com/robotology/walking-controllers/pull/144, this would fit well in a separate file named ``navigation``